engine-tests/src/utils/workspace.rs

```diff
@@ -236,6 +238,40 @@ pub async fn transfer_nep_141(
     Ok(())
 }
 
+pub async fn transfer_call_nep_141(
```

Comment:
I believe it should be annotated with `#[cfg(not(feature = "ext-connector"))]`

Reply:
219007bf9b048c152dd5f94abafd7126e9316331